action/protocol/staking/protocol.go

```diff
@@ -339,7 +339,7 @@ func (p *Protocol) handle(ctx context.Context, act action.Action, csm CandidateS
 	if receiptErr, ok := err.(ReceiptError); ok {
 		actionCtx := protocol.MustGetActionCtx(ctx)
 		log.L().With(
-			zap.String("actionHash", hex.EncodeToString(actionCtx.ActionHash[:]))).Info("Failed to commit staking action", zap.Error(err))
+			zap.String("actionHash", hex.EncodeToString(actionCtx.ActionHash[:]))).Warn("Failed to commit staking action", zap.Error(err))
```

Comment:
can you also change the "failed to get committed state" to DEBUG?

Reply:
also, WARN is actually higher level than info, if want to mute need to change to DEBUG?